src/main.js

```diff
@@ -29,6 +29,7 @@ viewSavedCoversButton.addEventListener('click', changeToSavedCovers);
 homeButton.addEventListener('click', goHome)
 makeMyBook.addEventListener('click', makeBookCover)
 saveCoverButton.addEventListener('click', saveCoverData)
+savedCoverViewer.addEventListener('click', deleteCover)
```

Comment:
This is the code in question to remove for now.

Reply:
Will do!

Reply:
Great! Resolved.